WI_FAIL_FAST_ASSERT missing semicolon

This line:
WI_FAIL_FAST_ASSERT(::GetTickCount64() == 0);

Gives this error:
error C2143: syntax error: missing ';' before '}'

Because this line:
__RFF_FN(FailFast_Unexpected)(__RFF_INFO_ONLY(#condition)) \

Should have a semicolon:
__RFF_FN(FailFast_Unexpected)(__RFF_INFO_ONLY(#condition)); \

--- a/include/wil/result_macros.h
+++ b/include/wil/result_macros.h
@@ -1290,7 +1290,7 @@ WI_ODR_PRAGMA("WIL_FreeMemory", "0")
         if (!wil::verify_bool(condition)) \
         { \
             WI_ASSERT_FAIL(#condition); \
-            __RFF_FN(FailFast_Unexpected)(__RFF_INFO_ONLY(#condition)) \
+            __RFF_FN(FailFast_Unexpected)(__RFF_INFO_ONLY(#condition)); \
         } \
     } while (0, 0)
 #define WI_FAIL_FAST_ASSERT_MSG(condition, msg) \

